modele-social/règles/dirigeant.yaml

```diff
@@ -75,6 +75,10 @@ dirigeant . assimilé salarié . réduction ACRE . notification taux annuel:
 
 dirigeant . auto-entrepreneur:
   rend non applicable: contrat salarié
+  remplace: 
+    règle: entreprise . imposition
+    # TODO devrait remplacer par l'option micro-BNC
```

Comment:
Oups, je suis allé un peu vite !

Une remarque : "IR" comme "imposition de l'entreprise" ça veut toujours dire BNC / BIC je pense

Reply:
Oui, c'était dans l'idée de séparer Micro-* / IR / IS comme options possibles côté fiscal.